### DISCUSSION TITLE
Version 1.0.5

### DISCUSSION THREAD

**Discussion by donavanbecker** [homebridge-plugins/homebridge-august#17] [Announcements]
## What's Changes
- Fixed issue with RetryCount showing false `LockCurrentState` of `JAMMED`.

**Full Changelog**: https://github.com/donavanbecker/homebridge-august/compare/v1.0.4...v1.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/donavanbecker/homebridge-august/releases/tag/v1.0.5'>Version 1.0.5</a>.</em>

**Comment by Geebs11** [comment]
Hello, I am trying to an August WiFi lock via Homebridge, and have installed your August Plugin and after trying various Node.js versions (14.18.0 suggested didn't work but the Homebridge Logs stated than I needed 14.20.1 which got me closer. Now I have one remaining Error message:

[August] Discover Devices: TypeError: Cannot convert undefined or null to object

and I'd really like to know what I've got wrong.

I have installed Nest & MyQ Plugins successfully.

Sorry if this is the wrong vehicle for Questions...a newbie to this.

Thanks for anything you can suggest to get my August WiFi lock installed via Homebridge.

Gary

**Comment by Muz-Ahmed** [comment]
Hi, I am trying to get a Yale Keyless Connected lock to work with Homebridge, I have installed your August Plugin and it has successfully connected but I keep getting this error: `[11/23/2022, 11:31:58 PM] [August] Discover Devices: TypeError: Cannot convert undefined or null to object
Session validated!`.

I assume it’s because the lock is not supported by this plugin but is there a way to get it to work or to be supported? I know Yale and August are pretty much the same thing, I can even use the August App to control my lock the same way with the Yale app.

I’m using the Yale Access Module with the Yale Connect WiFi Bridge to give me the app access.

I have installed and authenticated my account properly it’s just the issue with getting the plugin to support my lock.Iif there’s any help I could get in getting this to work - I’d appreciate it.

Thanks
